src/pages/feed.js

```diff
@@ -82,62 +72,50 @@ export default class Feed extends Component {
     }
   }
 
-  render() {
-    const info = (index) => (
-      <div style={{ display: 'inline' }}>
-        <a
-          style={{
-            color: '#000',
-            '&:hover': {
-              color: '#000',
-            },
-          }}
-          href={`/objkt/${this.state.items[index].token_id}`}
-        >
-          OBJKT#{this.state.items[index].token_id}
-        </a>
-      </div>
-    )
-    const image = (index) => (
-      <div>
-        <LazyLoadImage
-          className="media"
-          style={{ maxHeight: '50vh', height: 'auto', width: 'auto' }}
-          src={`https://ipfs.io/ipfs/${
-            this.state.items[index].token_info.artifactUri.split('//')[1]
-          }`}
-          alt="💥"
-        />
-      </div>
-    )
+  setActiveTab = (tab) => {
+    console.log('yoo')
```

Comment:
👋 Yo! but remove console log :)

Reply:
:P will do!

Reply:
@andrevenancio now it's back in it's proper state :)